workers/unity/Assets/Gdk/Core/Systems/CustomSpatialOSSendSystem.cs

```diff
@@ -16,7 +16,7 @@ protected override void OnCreateManager(int capacity)
             worker = WorkerRegistry.GetWorkerForWorld(World);
 
             spatialOSSendSystem = World.GetOrCreateManager<SpatialOSSendSystem>();
-            spatialOSSendSystem.RegisterCustomReplicationSystem(typeof(T));
+            spatialOSSendSystem.TryRegisterCustomReplicationSystem(typeof(T));
```

Comment:
Not checking the return value? Shouldn't the error handling for this be outside of `TryRegister...`?
If something starts with `Try`, I wouldn't expect it to do any error reporting, and to leave it up to the caller.